packages/base/any/kernels/5.10-lts/patches/0043-switchdev-prestera-v3.1.1.patch

```diff
@@ -0,0 +1,11812 @@
+diff --git a/drivers/net/ethernet/marvell/prestera/Makefile b/drivers/net/ethernet/marvell/prestera/Makefile
```

Comment:
Please add a proper commit message and format the patch five with `git format-patch`.

Reply:
done